charts/alidns-webhook/templates/pki.yaml

```diff
@@ -29,7 +29,7 @@ metadata:
     heritage: {{ .Release.Service }}
 spec:
   secretName: {{ include "alidns-webhook.rootCACertificate" . }}
-  duration: 43800h # 5y
+  duration: 43800h0m0s # 5y
```

Comment:
is there a particular reason to explicitly set minutes and seconds to zero ?

Reply:
Actually the api-server itself modifies it to `43800h0m0s` value, and it is not necessary to explicitly set, but it's better to have it explicitly as Gitops tools such as `Argocd` and `Flux` will consider this as `Out of Sync` since the object in ETCD is different from the object in git. So there is always warning about this.

Reply:
Also the worse case is if someone has autoSync enabled, it would be an infinite loop of syncs about this, unless one configures the setting of the chart in Arogocd to ignore this change (which is not good as in that case, it will consider it sync even if the value is `1h`, as it ignores it).